politeiawww/legacy/invoices.go

```diff
@@ -2535,3 +2535,38 @@ func (p *Politeiawww) processProposalBillingDetails(pbd cms.ProposalBillingDetai
 	reply.Details = spendingSummary
 	return reply, nil
 }
+
+func (p *Politeiawww) processProposalBilledState(pbd cms.ProposalBillingDetails) (*cms.RecordBilledStateReply, error) {
+	propInvoices, err := p.cmsDB.InvoicesByLineItemsProposalToken(pbd.Token)
+	if err != nil {
+		return nil, err
+	}
+	reply := &cms.RecordBilledStateReply{
+		PaidInvoices: make([]cms.PublicPaidInvoiceRecord, 0),
+	}
+
+	totalSpent := int64(0)
+	for _, dbInv := range propInvoices {
+		u, err := p.db.UserGetByPubKey(dbInv.PublicKey)
+		if err != nil {
+			log.Errorf("getUserByPubKey: token:%v "+
+				"pubKey:%v err:%v", dbInv.PublicKey, err)
+		} else {
+			dbInv.Username = u.Username
+		}
```

Comment:
any reason why we need to do this? seems this is the only place where u is used, can remove this block of code if we really do not need the username in calculating payout.